js/ui/calendar/ui.calendar.js

```diff
@@ -395,11 +417,39 @@ const Calendar = Editor.inherit({
 
     _init: function() {
         this.callBase();
+        this._initSelectionStrategy();
         this._correctZoomLevel();
         this._initCurrentDate();
         this._initActions();
     },
 
+    _initSelectionStrategy: function() {
+        const strategyName = this._getSelectionStrategyName();
+        const strategy = SELECTION_STRATEGY_CLASSES[strategyName];
+
+        if(!(this._selectionStrategy && this._selectionStrategy.NAME === strategyName)) {
```

Comment:
Isn't it more readable, how do u think?
```suggestion
        if(!this._selectionStrategy || this._selectionStrategy.NAME !== strategyName)) {
```

Reply:
a bit, fixed